client/app/scripts/superdesk/notification/notification.js

```diff
@@ -22,6 +34,9 @@
         ws.onmessage = function(event) {
             var msg = angular.fromJson(event.data);
             $rootScope.$broadcast(msg.event, msg.extra);
+            if (_.has(ReloadEvents, msg.event)) {
+                $rootScope.$broadcast('reload', msg);
```

Comment:
actually, we tried here injecting reloadService but , lot of tests (more than 20) was starts failing, if you notice ozan's commit above as well for that prpose, so after that come up that solution.